android/src/main/java/com/zmxv/RNSound/RNSoundModule.java

```diff
@@ -44,6 +44,13 @@ public RNSoundModule(ReactApplicationContext context) {
     this.category = null;
   }
 
+  private void setOnPlay(boolean isPlaying) {
```

Comment:
this file already has `sendEvent()` method!! Inside `setOnPlay` method,  I'm using `sendEvent()` method to emit events. I wrapped calling `sendEvent()` and making parameters with `setOnPlay` method to avoid redundacncy

Reply:
Oh, you are right. 

I have one little more feedback. Since it wraps around `sendEvent` function, I think `sendOnPlayChangeEvent` or `sendOnPlayEvent` sounds more natural. How do you think?